src/pwncore/models/ctf.py

```diff
@@ -48,7 +48,9 @@ class Meta:
 
 class ViewedHint(Model):
     team: fields.ForeignKeyRelation[Team] = fields.ForeignKeyField("models.Team")
-    hint: fields.ForeignKeyRelation[Hint] = fields.ForeignKeyField("models.Hint")
+    hint: fields.ForeignKeyRelation[Hint] = fields.ForeignKeyField(
+        "models.Hint", related_name="hints"
```

Comment:
why is related_name added here?
Why is Hint.hints pointing to ViewedHint records?
Remove if not intended